botocore/endpoint.py

```diff
@@ -224,17 +224,15 @@ def _get_proxies(url):
 def get_endpoint(service, region_name, endpoint_url, verify=None):
     service_name = getattr(service, 'signing_name', service.endpoint_prefix)
     endpoint_prefix = service.endpoint_prefix
-    signature_version = getattr(service, 'signature_version', None)
     session = service.session
     event_emitter = session.get_component('event_emitter')
     user_agent = session.user_agent()
     return get_endpoint_complex(service_name, endpoint_prefix,
-                                signature_version,
                                 region_name, endpoint_url, verify, user_agent,
                                 event_emitter)
 
 
-def get_endpoint_complex(service_name, endpoint_prefix, signature_version,
+def get_endpoint_complex(service_name, endpoint_prefix,
```

Comment:
This is a backward-incompatible change of a public function. It's possible we might break someone using Botocore.